docs/contributing/client-asset-tunnelling.md

```diff
@@ -0,0 +1,40 @@
+---
+id: client-asset-tunnelling
+title: Client Asset Tunnelling
+---
+
+It's often useful to test local editor/client changes against the production
+backend. The editor includes a "Tunnel" feature to support this.
```

Comment:
I'd suggest something like:

"The best way to work on the Dark client is by building it against the production Dark APIs. In the future as we move more part of the editor to be built in Dark, it will become the only way to work on the Dark client".